projects/demo-integrations/cypress/support/commands/basic.ts

```diff
@@ -74,6 +76,12 @@ Cypress.Commands.add(
     tuiScrollIntoView,
 );
 
+Cypress.Commands.add(
+    `tuiHover`,
+    {prevSubject: [`optional`, `element`, `window`, `document`]},
```

Comment:
I think only `element` is valid option 🤔

Reply:
fixed